### PR TITLE
V2.1.1-dev

### PR DESCRIPTION
**Pull request by felixhao28**

### CODE REVIEW

**Comment by felixhao28** [comment]
@maliayas I still don't have write access. Can you confirm it?

**Comment by maliayas** [comment]
Sorry for that. How about now?

GitHub's organization feature seems a little complicated.

File: Default (Windows).sublime-keymap [hunk 1]
```diff
@@ -441,7 +441,6 @@
     // run paste as link command on selected text
     { "keys": ["ctrl+super+v"], "command": "paste_as_link", "context":
         [
-            { "key": "selection_empty", "operator": "equal", "operand": false, "match_all": true },
```
Comment (maliayas) [inline Default (Windows).sublime-keymap]: Do we need to apply this to other keymaps as well?

File: tests/Markdown.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ __bold__
 
 **bold\***
 
+___bold_and_italic___
+
+***bold_and_italic***
+
```
Comment (maliayas) [inline tests/Markdown.md]: :+1:

File: Default (Windows).sublime-keymap (hunk 1 above)
Comment (felixhao28) [inline Default (Windows).sublime-keymap]: You are right! Fixed here 6d8eff80b0d6c8608fe3a3023f99e081426dfb48 .

**Comment by felixhao28** [comment]
Thanks. I have merged my pr to v2.1.1 branch.

2015-03-07 8:26 GMT+08:00 Ali Ayas <notifications@github.com>:

> Sorry for that. How about now?
>
> GitHub's organization feature seems a little complicated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SublimeText-Markdown/MarkdownEditing/pull/240#issuecomment-77661064>
> .
>



-- 
Felix Hao

Washizaki's Lab
Computer Science Department
Graduate School of Fundamental Science and Engineering
Waseda University

&

Innovation Engineering Group
Microsoft Research Asia
Beijing, China

**Pull request merged**